<?xml version="1.0"?>
<launch>



	<include file="$(find cob_script_server)/launch/script_server.launch" />

	<node pkg="joy" type="joy_node" name="joy" >
		<param name="deadzone" value="0.12" />
	</node>

	<node pkg="cob_teleop_cob4" type="cob_teleop_cob4" name="cob_teleop_cob4" output="screen" >
		<remap from="base_controller_command" to="base_controller/command"/>
		<remap from="sensorring_controller_command" to="sensorring_controller/command_twist"/>
		<remap from="torso_controller_command" to="torso_controller/command_twist"/>
		<remap from="arm_joint_left" to="arm_left_controller/command_vel"/>
		<remap from="arm_joint_right" to="arm_right_controller/command_vel"/>
		<remap from="head_controller_command" to="head_controller/command_twist"/>
		<remap from="arm_cart_left" to="arm_left_controller/command_twist"/>
		<remap from="arm_cart_right" to="arm_right_controller/command_twist"/>
		<rosparam command="load" file="$(find cob_teleop_cob4)/cfg/arm_uri.yaml" />
		<rosparam command="load" file="$(find cob_teleop_cob4)/cfg/component.yaml" />
	</node>

</launch>
